cmd/kubeadm/app/cmd/token.go

```diff
@@ -61,22 +64,27 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 		},
 	}
 
-	var token string
+	var usages []string
 	var tokenDuration time.Duration
 	createCmd := &cobra.Command{
-		Use:   "create",
+		Use:   "create [token]",
 		Short: "Create bootstrap tokens on the server.",
 		Run: func(tokenCmd *cobra.Command, args []string) {
-			err := RunCreateToken(out, tokenCmd, tokenDuration, token)
+			token := ""
+			if len(args) != 0 {
+				token = args[0]
+			}
+			client, err := kubemaster.CreateClientFromFile(getKubeConfigPath())
+			kubeadmutil.CheckErr(err)
+
+			err = RunCreateToken(out, client, token, tokenDuration, usages)
 			kubeadmutil.CheckErr(err)
 		},
 	}
 	createCmd.PersistentFlags().DurationVar(&tokenDuration,
 		"ttl", kubeadmconstants.DefaultTokenDuration, "The duration before the token is automatically deleted. 0 means 'never expires'.")
-	createCmd.PersistentFlags().StringVar(
-		&token, "token", "",
-		"Shared secret used to secure cluster bootstrap. If none is provided, one will be generated for you.",
-	)
+	createCmd.PersistentFlags().StringSliceVar(&usages,
+		"usages", kubeadmconstants.DefaultTokenUsages, "The functions this token can be used for.")
```

Comment:
fair enough! I just thought the plural of it sounded strange. But, if there is support, totally fine.